pkg/data/data.go

```diff
@@ -1 +1,26 @@
 package data
+
+import (
+	"embed"
+	"io/fs"
+	"sort"
+)
+
+//go:embed embed/*
+var embedFS embed.FS
+
+func AssetNames() []string {
+	var out []string
+	fs.WalkDir(embedFS, ".", func(path string, entry fs.DirEntry, err error) error {
+		if entry.Type().IsRegular() {
+			out = append(out, path[6:])
+		}
+		return nil
+	})
+	sort.Strings(out)
+	return out
+}
+
+func Asset(name string) ([]byte, error) {
+	return embedFS.ReadFile("embed/" + name)
+}
```

Comment:
These two functions end up copy-pasted in a bunch of places, I wonder if there is an easy way to make them reusable...